azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -353,4 +353,7 @@ ExecutableRampExceptionalJobItemsMap fetchExecutableRampExceptionalJobItemsMap()
       throws ExecutorManagerException;
 
   void updateExecutableRamp(ExecutableRamp executableRamp) throws ExecutorManagerException;
+
+  int updateVersionSetId(final int executionId, final int versionSetId)
```

Comment:
Can you please add javadoc comment for this method?

Reply:
Done.